Source/WTF/wtf/CheckedPtr.h

```diff
@@ -264,6 +264,7 @@ class CheckedPtr {
 template <typename T, typename PtrTraits>
 struct GetPtrHelper<CheckedPtr<T, PtrTraits>> {
     typedef T* PtrType;
+    typedef T UnderlyingType;
```

Comment:
In new code we should use `using` rather than `typedef`. Noticed this many places in this patch.

Reply:
Fixed.